tutorials/convert_lit_models.md

```diff
@@ -0,0 +1,40 @@
+## Converting a Lit-GPT Formatted Model to its Original Format
+
+Several models retrieved from external sources must be reformatted with naming conventions for Lit-GPT weights before finetuning. We've provided a helpful script to convert models finetuned with Lit-GPT back to their original format.
+
+The way the script is used depends on the finetuning method. The commands for converting a model back to its original format after using one of the several finetuning methods supported by Lit-GPT are shown below.
+
+> **Note**
```

Comment:
The "Note" notation doesn't work with line breaks unfortunately. If you look at the rendered markdown, you'll see it appears as a "quote"

Reply:
seems this admonition style broke in the last 24 hours: https://github.com/orgs/community/discussions/16925

Reply:
I commited a change to remove the space and line break - which works here, but not in the rendered .md file

> **Note**
> each example shown below uses Falcon-7B please be sure to update --checkpoint_dir accordingly

Reply:
the new syntax for .md file is

```md
> [!NOTE]  
> ipsum lorem
```

> [!NOTE]  
> ipsum lorem

Reply:
@JustinGoheen Could you fix the rest of the usages in this repo in a separate PR?

Reply:
will do. see #305